composer.json

```diff
@@ -23,6 +23,11 @@
             "chobie\\" : "src"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "chobie\\Tests\\": "tests"
```

Comment:
Nice. While you're at it please namespace and rename `Jira_Api_Authentication_BasicTest` class.